pkg/reconciler/build/build.go

```diff
@@ -110,8 +111,14 @@ func (c *Reconciler) reconcile(ctx context.Context, build *buildapi.Build) error
 			return err
 		}
 
+		cacheImageId, err := c.MetadataRetriever.GetCacheImage(ctx, build)
```

Comment:
It would be interesting to see if we could acquire this from `report.toml`. Perhaps we should create a cnb rfc?

Reply:
When comparing it to the actual logs of the image pushed to the registry, it is somehow the wrong `sha256` and the content of `LastCacheImage` can't be pulled. So we should look into that!!!